src/sentry/conf/server.py

```diff
@@ -117,8 +118,13 @@ def env(key, default="", type=None):
     os.path.join(PROJECT_ROOT, os.pardir, os.pardir, "config", "relay")
 )
 
+if platform.system() == "Linux":
+    nginx_conf_file = "nginx.linux.conf"
+else:
+    nginx_conf_file = "nginx.conf"
+
 REVERSE_PROXY_CONFIG = os.path.normpath(
-    os.path.join(PROJECT_ROOT, os.pardir, os.pardir, "config", "reverse_proxy", "nginx.conf")
+    os.path.join(PROJECT_ROOT, os.pardir, os.pardir, "config", "reverse_proxy", "nginx_conf_file")
```

Comment:
This should be using nginx_conf_file the variable, not "nginx_conf_file" the string.

Reply:
I'll fix soon, but I'm on another branch atm and don't feel like it right now.